### DISCUSSION TITLE
Edit a page

### DISCUSSION THREAD

**Discussion by haf** [swellstores/community#73] [Help] [answered]
Hi,

My site that uses Vercel Commerce's frontend, has a page /about, but I can't find where in the UI to edit the contents of this page?

**Comment by maarten2424** [comment]
Hey @haf the Vercel Commerce solution is not maintained by us, but I might help. Did you create the route yourself? Or is it a dynamic route, e.g. `[...pages].tsx`
If it's a dynamic route, Vercel Commerce is querying our API and you should be able to edit the content of the about page directly trough the dashboard storefront.

**Comment by haf** [comment]
It's querying your API but I cannot for the life of me find where in your dashboard storefront I can edit the pages?

**Comment by haf** [comment, marked as answer]
You can't edit the pages unless you keep the default theme.

**Reply by ericingram** [reply]
We're working on a solution for that right now! Content editing was originally part of our theme editor which doesn't (yet) work with externally hosted frontends.

An enhancement to enable editing content outside of the theme editor, should be out sometime in the next few months.
